packages/core/src/components/Dialog/Dialog.tsx

```diff
@@ -544,98 +549,99 @@ export default class Dialog extends PureComponent<DialogProps, DialogState> {
             );
           }}
         </Reference>
-        {createPortal(
-          <Popper
-            placement={position as unknown as PopperJS.Placement}
-            modifiers={[
-              {
-                name: "offset",
-                options: {
-                  offset: [moveBy.secondary, moveBy.main]
-                }
-              },
-              {
-                name: "zIndex",
-                enabled: true,
-                phase: "write",
-                fn({ state }) {
-                  if (zIndex) {
-                    state.styles.popper.zIndex = String(zIndex);
+        {isClient() &&
```

Comment:
In SSR it wouldn't render anything? is it intentional?

Reply:
I don't have a dom element to send to the createPortal so I guess I can't